app/models/manageiq/providers/physical_infra_manager.rb

```diff
@@ -17,6 +17,15 @@ class PhysicalInfraManager < BaseManager
     virtual_column :total_hosts, :type => :integer
     virtual_column :total_vms,   :type => :integer
 
+    virtual_column :total_valid, :type => :integer
+    virtual_column :total_warning, :type => :integer
+    virtual_column :total_critical, :type => :integer
+    virtual_column :health_state_info, :type => :string
```

Comment:
@gtanzillo The intention is to add it to the physical infra provider JSON response at the ManageIQ API. I put it as a `:string` because there is no type `:hash` for virtual columns. What I could do is do create these columns as `:json`. What do you think of that? It's more accurate, and fit to the purpose, then `:string`.

Reply:
@EsdrasVP Yes, I think that the `:json` type makes more sense here and I would go with that.

Reply:
@gtanzillo Great, I made the change, so `:json` will be used instead of `:string`.